networkx/algorithms/approximation/broadcasting.py

```diff
@@ -0,0 +1,98 @@
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = [
+    "tree_broadcast",
+]
+
+
+def update_broadcast_label(G, U, v):
+    adj = [u for u in G.neighbors(v) if u in U]
+    adj.sort(key=lambda u: G.nodes[u]["value"], reverse=True)
+    return max([G.nodes[u]["value"] + i for i, u in enumerate(adj, start=1)])
```

Comment:
Use a leading `_` underscore to indicate a "helper" function that won't be made public by being put into the `__all__` variable.  Also, since this one is only 2 lines long, you might consider in-lining it to help the reader understand the update.  Actually, it doesn't look like this updates anything. It returns a maximal value. So it at least needs a new name -- or else to be in-lined.
```suggestion
def _update_broadcast_label(G, U, v):
    adj = sorted(G[v] & U, key=values.get, reverse=True)
    return max(values[u] + i for i, u in enumerate(adj, start=1))
```

Reply:
ok, I've made the changes you required. I think renaming it to ``_get_max_broadcast_value`` is a bit more suitable.